src/float.rs

```diff
@@ -1889,7 +1889,7 @@ macro_rules! float_impl_std {
 
 #[cfg(all(not(feature = "std"), feature = "libm"))]
 macro_rules! float_impl_libm {
-    ($T:ident $decode:ident $LibmImpl:ident) => {
+    ($T:ident $decode:ident { $($LibmImpl:tt)* }) => {
```

Comment:
I think rather than `tt`-munching, it would be nicer to now flip the macro around and just fill in the common methods. Something like:

```rust
impl Float for f32 {
    float_impl_libm!(...) // common stuff
    // then f32-specific items
}
impl Float for f64 {
    float_impl_libm!(...) // common stuff
    // then f64-specific items
}

Reply:
Alright, I inverted it.